incubator/kitura/README.md

```diff
@@ -0,0 +1,126 @@
+# Kitura Stack
+
+The [Kitura](https://kitura.io) stack extends the [Swift stack](../swift/README.md).  Kitura is an open source web framework for server-side Swift that can be used to build web applications and REST APIs, with full support for databases, WebSockets, OpenAPI and much more.
+
+## Templates
+
+Templates are used to create your local project and start your development. When initializing your project you will be provided with the default template project. This template provides a simple application that hosts the Kitura landing page, and an example route on `/hello` which returns a simple greeting.
```

Comment:
"When initializing your project, you are provided" - preference for writing in the present tense where possible.

"and an example route on `/hello`, which" - can you add a comma in here please?